automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/NotebookPyKernelSpec.scala

```diff
@@ -63,6 +63,18 @@ class NotebookPyKernelSpec extends ClusterFixtureSpec {
       }
     }
 
+    "should update dateAccessed if the notebook is open" in { clusterFixture =>
+      withWebDriver { implicit driver =>
+        withNewNotebook(clusterFixture.cluster) { notebookPage =>
+          val firstApiCall = Leonardo.cluster.get(clusterFixture.billingProject, clusterFixture.cluster.clusterName)
+          println(firstApiCall.dateAccessed)
```

Comment:
Use `log.info` instead of `println`.

Also log a message like "Sleeping for 90 seconds to simulate an idle notebook...". It helps when reading/debugging test logs.